.github/workflows/continuous-deployment.yml

```diff
@@ -485,3 +485,7 @@ jobs:
         run: |
           name="gha-agent-${{ env.stackName }}"
           ops/helper-scripts/dev-rm-allowed-ip.sh ${{ steps.rgApp.outputs.out }} ${{ env.stackName }} ${name:0:32}
+      - name: Enable access
+        run: |
+          az webapp config access-restriction add --resource-group ${{ steps.rgApp.outputs.out }} --name ${{ env.webappName }} --rule-name AllowAll --action Allow --ip-address 0.0.0.0/0 --priority 100 1>/dev/null || true
+          az functionapp config access-restriction add --resource-group ${{ steps.rgApp.outputs.out }} --name ${{ env.apiName }} --rule-name AllowAll --action Allow --ip-address 0.0.0.0/0 --priority 100 1>/dev/null || true
```

Comment:
- I agree, we can improve on this by implementing this step as a separate job. It would require ensuring the new job has all the values it needs to be passed in as inputs. As we may add more jobs, we may want to consider decomposing continous-deployment into several workflows later on.
- Since we are just adding a rule to allow all ips it should not impact a running integration test if the implementation is left as is.
- I was trying to keep existing operations in tack at the moment to keep things close to how it can be executed in the USTP environment (if post deployment tests will be executed)